core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -25,6 +25,7 @@
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicLong;
 
+import com.microsoft.applicationinsights.channel.concrete.nop.NopTelemetryChannel;
```

Comment:
It doesn't look like this is used anywhere in this file

Reply:
correct. fixed